set.mm

```diff
@@ -237961,6 +238019,22 @@ by a normal subgroup (resp. two-sided ideal).  (Contributed by Mario
   brgici $p |- ( F e. ( R GrpIso S ) -> R ~=g S ) $=
     ( cgim co wcel c0 wne cgic wbr ne0i brgic sylibr ) CABDEZFNGHABIJNCKABLM $.
 
+  ${
+    gim0to0.a $e |- A = ( Base ` R ) $.
+    gim0to0.b $e |- B = ( Base ` S ) $.
+    gim0to0.n $e |- N = ( 0g ` S ) $.
+    gim0to0.0 $e |- .0. = ( 0g ` R ) $.
+    $( A group isomorphism maps the zero of one group (and only the zero) to
+       the zero of the other group.  (Contributed by AV, 24-Oct-2019.)
+       (Revised by Thierry Arnoux, 23-May-2023.) $)
+    gim0to0 $p |- ( ( F e. ( R GrpIso S ) /\ X e. A )
+                    -> ( ( F ` X ) = N <-> X = .0. ) ) $=
+      ( cgim co wcel wa cghm wf1 wceq syl w3a cfv wb gimghm gimf1o f1of1 anim1i
+      wf1o jca df-3an sylibr f1ghm0to0 ) ECDMNOZGAOZPZECDQNOZABERZUNUAZGEUBFSGH
+      SUCUOUPUQPZUNPURUMUSUNUMUPUQCDEUDUMABEUHUQABCDEIJUEABEUFTUIUGUPUQUNUJUKAB
+      CDEHGFIJLKULT $.
+  $}
+
```

Comment:
(optional) maybe move before brgic

Reply:
Good suggestion, because starting with ~brgic, there are theorems for a new symbol ` ~=g `.